ci-operator/config/openshift/csi-driver-shared-resource/openshift-csi-driver-shared-resource-master.yaml

```diff
@@ -3,6 +3,14 @@ base_images:
     name: "4.10"
     namespace: ocp
     tag: base
+  must_gather:
+    name: "4.10"
+    namespace: ocp
+    tag: must-gather
+  ocp_4.10_must-gather:
+    name: "4.10"
+    namespace: ocp
+    tag: must-gather
```

Comment:
what's the motivation for referencing/aliasing the same image twice?

Reply:
I monkey saw / monkey did wrt what the other features did.  For example, https://github.com/openshift/release/blob/master/ci-operator/config/openshift/local-storage-operator/openshift-local-storage-operator-master.yaml

Reply:
but certainly I can entertain optimizations if desired

Reply:
@petr-muller is there some reason you're aware of for creating+referencing both aliases(pointing to the same underlying tag) in this way?  It seems odd to me, but i haven't kept up w/ the latest CI schemas.

Reply:
Our automation does these for Dockerfiles that contain pullspecs to registry.ci.openshift.org, so that during the build, the local image is used instead of the registry.ci one - the latter would work but is less efficient. 

You can leave these out when setting up configs. They are only needed if your Dockerfiles have such pullspecs. And if they do, the automation will create these replacements automatically.